pages/home/configure/engine.mdx

```diff
@@ -164,15 +162,21 @@ The Redis-based token store is persistent and can be used in production environm
 
 The `auth.providers` section defines the providers that users can authorize with. Arcade Engine supports many built-in [auth providers](/integrations/auth), and can also connect to any [OAuth 2.0](/integrations/auth/oauth2)-compatible authorization server.
 
-The `providers` array contains provider definitions, each with an `id` and provider-specific configuration:
+The `providers` array contains provider definitions, each which must have a unique `id` within this Arcade Engine instance. There are two ways to configure a provider:
+
+1. For [built-in providers](/integrations/auth), use the `provider_id` field to reference them by name. For example:
```

Comment:
I think auth providers were moved from `/integrations/auth` to `/home/auth-providers`

Reply:
Thanks! Updated all the links in [1f08c0f](https://github.com/ArcadeAI/docs/pull/109/commits/1f08c0f11a73935c2aea2ddd47c3135059a34a8f)